x-pack/plugins/observability_solution/entity_manager/server/lib/entities/helpers/generate_index_name.ts

```diff
@@ -7,14 +7,16 @@
 
 import { EntityDefinition } from '@kbn/entities-schema';
 import {
-  ENTITY_HISTORY_BASE_PREFIX,
-  ENTITY_LATEST_BASE_PREFIX,
-} from '../../../../common/constants_entities';
+  ENTITY_INDEX_PREFIX,
+  ENTITY_SCHEMA_VERSION_V1,
+  ENTITY_LATEST,
+  ENTITY_HISTORY,
+} from '@kbn/entityManager-plugin/common/constants_entities';
 
 export function generateLatestIndexName(definition: EntityDefinition) {
-  return `${ENTITY_LATEST_BASE_PREFIX}.${definition.id}`;
+  return `${ENTITY_INDEX_PREFIX}-${ENTITY_SCHEMA_VERSION_V1}-${ENTITY_LATEST}-${definition.id}`;
```

Comment:
i think i prefer having the prefix defined in the constants file (both for index names and transforms/ingest pipelines) - it makes them more visible in the context of the constants use to build them IMO. 

it also answers the question about passing the schema version - i.e. it is no longer possible.

Reply:
Sounds good, fixed in https://github.com/elastic/kibana/pull/187507/commits/a54e47d68d98f7dee6502d9906ae1cf408a4967b